src/plugins/serverless/config/es.yml

```diff
@@ -0,0 +1,9 @@
+# Elasticsearch Serverless configuration
```

Comment:
nit: I wonder if the configs could be placed in a package instead. The plugin doesn't seem to do anything.